src/loaders/api/api.js

```diff
@@ -2,20 +2,23 @@
  * Copyright 2020 New Relic Corporation. All rights reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-import slice from 'lodash._slice'
 import { FEATURE_NAMES } from '../features/features'
 import { getRuntime, setInfo, getInfo } from '../../common/config/config'
 import { handle } from '../../common/event-emitter/handle'
-import { mapOwn } from '../../common/util/map-own'
 import { ee } from '../../common/event-emitter/contextual-ee'
 import { now } from '../../common/timing/now'
 import { drain, registerDrain } from '../../common/drain/drain'
 import { onWindowLoad } from '../../common/window/load'
-import { isWorkerScope } from '../../common/util/global-scope'
+import { isBrowserScope, isWorkerScope } from '../../common/util/global-scope'
 import { warn } from '../../common/util/console'
 import { SUPPORTABILITY_METRIC_CHANNEL } from '../../features/metrics/constants'
+import { gosCDN } from '../../common/window/nreum'
+import { putInBrowserStorage, removeFromBrowserStorage } from '../../common/window/session-storage'
 
-function setTopLevelCallers (nr) {
+export const CUSTOM_ATTR_GROUP = 'CUSTOM/' // the subgroup items should be stored under in storage API
+
+export function setTopLevelCallers () {
```

Comment:
Is there a reason `setTopLevelCallers` got moved back to the API file? This fn keeps getting moved around PR to PR

Reply:
Yeah, aside from it being in the file still from the branch parent, that contains a list of API functions and would be more reasonably found in _api.js_ where all API-related setup lives.